data/mods/Magiclysm/enchanted/enchanted_tools.json

```diff
@@ -0,0 +1,66 @@
+[
+  {
+    "id": "copper_bracelet_manafocus_off",
+    "type": "ARMOR",
+    "name": "copper mana focusing bracelet",
+    "description": "An ornate copper bracelet covered in magical symbols, crafted and manipulated to act as a focus for magical powers when you are enchanting. This is the second most basic enchanting tool. Activate to say the command word and turn the bracelet into a pencil like tool with a sharp point that you can use to etch magical symbols into other materials.",
+    "weight": 30,
+    "volume": 0,
+    "price": 25000,
+    "material": [ "copper" ],
+    "symbol": "[",
+    "color": "light_gray",
+    "flags": [ "FANCY" ],
+    "use_action": [
+      {
+        "type": "transform",
+        "target": "copper_bracelet_manafocus_on",
+        "active": false,
+        "msg": "You say the command and your bracelet straightens out into a tool.",
+        "menu_text": "Switch tool mode",
+        "moves": 50
+      }
+    ]
+  },
+  {
+    "id": "copper_bracelet_manafocus_on",
+    "type": "TOOL",
+    "name": "copper mana focuser",
+    "description": "An ornate copper rod with a fine sharp point covered in magical symbols, roughly the size of a pencil.  Crafted and manipulated to act as a focus for magical powers when you are enchanting.  This is the second most basic enchanting tool, think of it as a magical screwdriver.  You use the sharp point to etch symbols and runes into the substance of what you are enchanting.",
+    "weight": 30,
+    "volume": 0,
+    "price": 25000,
+    "cutting": 4,
+    "to_hit": 1,
+    "material": [ "copper" ],
+    "symbol": "[",
+    "color": "light_gray",
+    "qualities": [ [ "MANA_FOCUS", 1 ], [ "CUT_FINE", 1 ] ],
+    "use_action": [
+      {
+        "type": "transform",
+        "target": "copper_bracelet_manafocus_off",
+        "active": false,
+        "msg": "You say the command and your focus turns back into a bracelet.",
+        "menu_text": "Switch to bracelet",
+        "moves": 50
+      }
+    ]
+  },
+  {
+    "type": "GENERIC",
+    "id": "mana_infuser",
+    "symbol": "*",
+    "color": "white",
+    "name": "basic mana infuser",
+    "description": "Formerly a piece of ceramic, it has been reshaped into a small crystal rod.  The internal crystaline structure has been manipulated so that it can infuse mana into other substances, and in small quantities, substances into one another via magic.  It is the most basic tool for enchanting, think of it as a makeshift magical hammer.",
```

Comment:
no need to mention how basic of a tool this is. and by hammer, do you mean you beat things with this ceramic rod?

Reply:
I intended that text the same as the other, the hammer and screwdriver analogy is an indicator that, as with doing a lot of things in the game, you will need more than one tool to get things done.  You would need one of these to do anything, even make your own mana focus.